ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -22,6 +22,7 @@ import recsys from './plugins/videojs-recsys/plugin';
 // import runAds from './ads';
 import videojs from 'video.js';
 const canAutoplay = require('./plugins/canAutoplay');
+import { NEW_LIVESTREAM_LIVE_API } from 'constants/livestream';
 
 require('@silvermine/videojs-chromecast')(videojs);
```

Comment:
I was able to "cast", but it never plays on the TV side.  On the PC side, "LIVE" cannot be pressed.  Can probably disable the cast button for now.
But lower priority.

<img width="778" alt="image" src="https://user-images.githubusercontent.com/64950861/155056417-833e74ad-0d62-4a87-828c-840448be4b34.png">

Reply:
Agreed, this should be hidden for now and didn't work for me either. Should hide embed too if we can't get that to work.